### ISSUE TITLE
About pretrained resnet101

### ISSUE THREAD

**Issue by gangweix** [shjung13/Standardized-max-logits#6]
if pretrained:
        # model.load_state_dict(model_zoo.load_url(model_urls['resnet101']))
        print("########### pretrained ##############")
        # model.load_state_dict(torch.load('./pretrained/resnet101-imagenet.pth', map_location="cpu"))
        mynn.forgiving_state_restore(model, torch.load('./pretrained/resnet101-imagenet.pth', map_location="cpu"))

In line 339 of Resnet.py, './pretrained/resnet101-imagenet.pth' is not given, how I get it?

**Comment by NazirNayal8** [comment]
I was also wondering the same and after some searching I found these weights:

http://sceneparsing.csail.mit.edu/model/pretrained_resnet/resnet101-imagenet.pth

They have the exact same file name so it is highly likely that they are taken from there, but the confirmation of the authors is still required to be safe.

**Comment by shjung13** [comment]
Hello, I am so sorry for the late reply.

Thank you for pointing out this!

We added instructions about how to download the pretrained resnet101 network.

We use the same network that @NazirNayal8 found.

Please let us know if you have any further questions :)

Thank you.
